test/JDBC/input/BABEL-2144-vu-cleanup.sql

```diff
@@ -0,0 +1,6 @@
+drop view babel_2144_vu_prepare_v1;
+go
+drop table babel_2144_vu_prepare_t1;
+go
+drop table babel_2144_vu_prepare_t2;
+go
```

Comment:
Nit: Add a new line at the end of each new file.

Reply:
Done!